Python/wrfrun/pyWPS.py

```diff
@@ -624,6 +753,7 @@ def checkFileList(filelist, ListDir, okdates, depth):
 def processTimesteps(myid, dates):
   
   # create process sub-folder
+  os.system('pwd')
```

Comment:
if this is just to print the current working directory, there is a command for that: os.getcwd(); I would avoid unnecessary system/shell calls.